netbox/ipam/api/views.py

```diff
@@ -294,7 +294,8 @@ def post(self, request, pk):
         # Assign addresses from the list of available IPs and copy VRF assignment from the parent
```

Comment:
change the decorator of this func by this:
```
    @swagger_auto_schema(
        request_body=serializers.IPMaskLengthSerializer,
        responses={201: serializers.IPAddressSerializer(many=True)}
    )
```

Reply:
done